custom_components/frigate/views.py

```diff
@@ -250,39 +246,26 @@ async def _async_validate_signed_manifest(self, request: web.Request) -> bool:
         """Validate the signature for the manifest of this segment."""
         hass = request.app[KEY_HASS]
         secret = hass.data.get(DATA_SIGN_SECRET)
-
-        if secret is None:
-            _LOGGER.debug("Missing secret.")
-            return False
-
         signature = request.query.get(SIGN_QUERY_PARAM)
 
         if signature is None:
-            _LOGGER.debug("Missing signature.")
+            _LOGGER.error("Missing authSig query parameter on VOD segment request.")
```

Comment:
Nit: `warning` probably, as the integration itself won't have an error condition as a result. Same below.